app/controllers/users/documents_controller.rb

```diff
@@ -23,38 +30,50 @@ def update
       end
     end
 
-    def cover
-      respond_to do |format|
-        format.html
-        format.pdf do
-          render pdf: '表紙',
-            layout: 'pdf',
-            encording: 'UTF-8',
-            page_size: 'A4'
-        end
-      end
-    end
-
     private
 
+    def set_documents
+      @documents = current_business.request_orders.find_by(uuid: params[:request_order_uuid]).documents.order(id: :asc)
+    end
+
     def set_document
       @document = current_business.request_orders.find_by(uuid: params[:request_order_uuid]).documents.find_by(uuid: params[:uuid])
     end
 
-    def set_cover_documents
-      @cover_documents = current_business.documents.document_type_cover
+    # 更新書類の判定
+    def documents_params(document)
```

Comment:
メソッド名は以下にしたらいかがでしょうか？メソッドの中でupdateしているので、こっちの方が何をしているメソッドなのか明確になりそうです！
```suggestion
    def update_document(document)
```